src/controllers/eventController.js

```diff
@@ -143,6 +144,43 @@ eventController.post('/like/:id', async (req, res) => {
     }
 });
 
+// Get by Month.
+eventController.get('/month/:year/:month', async (req, res) => {
+    // TODO: Later add errors for wrong parameters.
+    try {
+        const { year, month } = req.params;
+        const { startDate, endDate } = getMonthRange(parseInt(year), parseInt(month) - 1);
+
+
+        const events = await getByMonth(startDate, endDate);
+
+        res.status(200).json(events);
+        res.end();
+    } catch (error) {
+        res.status(400).json(error.message);
+        res.end();
+    }
+});
+
 module.exports = {
     eventController,
 };
+
+// TODO: Add it in utils later!
```

Comment:
Will we need this getMonthRange ? It’s even from 0 to 11

Reply:
Yes, with this function i parse the year and the mont into date and then i compare the dates in the DB and then I take em.